src/components/profile/AdminUserManager.tsx

```diff
@@ -0,0 +1,126 @@
+import * as React from "react";
```

Comment:
This file was only renamed and need not be reviewed (though the diff suggests otherwise).